src/components/PageTitle/PageTitle.test.tsx

```diff
@@ -15,4 +15,23 @@ describe('Page Title component', () => {
 
     await waitFor(() => expect(document.title).toContain('Test | thenewboston'));
   });
+
+  test('renders with default og meta tags', async () => {
+    render(
+      <HelmetProvider>
+        <PageTitle title="Home" />
+      </HelmetProvider>,
+    );
+
+    await waitFor(() => {
+      expect(document.title).toContain('Home | thenewboston');
+      expect((document.querySelector('[property="og:type"]') as HTMLMetaElement).content).toBe(defaultOgType);
+      expect((document.querySelector('[property="og:title"]') as HTMLMetaElement).content).toBe('Home | thenewboston');
+      expect((document.querySelector('[property="og:description"]') as HTMLMetaElement).content).toBe(
+        defaultOgDescription,
+      );
+      expect((document.querySelector('[property="og:url"]') as HTMLMetaElement).content).toBe(defaultOgUrl);
+      expect((document.querySelector('[property="og:image"]') as HTMLMetaElement).content).toBe(defaultOgImageUrl);
```

Comment:
There a function `toHaveTextContent()`. You can use that.

Reply:
I'm getting the og property `content` tho, not the `textContent`

Reply:
🤦 oh yeah, my bad